extensions/extensions-encryption/aissemble-extensions-encryption-vault-python/tests/features/encryption.feature

```diff
@@ -1,32 +1,8 @@
 @properties
 Feature: Encryption
 
-  @integration
-  Scenario: Data can be encrypted and decrypted using Vault
```

Comment:
I: Move ITs to a separate feature file, so we can use feature tags to determine whether a Vault container is required.